test/fixtures/dialects/bigquery/select.sql

```diff
@@ -0,0 +1 @@
+SELECT 'metadata' AS key from foo;
```

Comment:
I personally like more generic files like this to dump a bunch of random test cases into over time instead of all the little ones. Some other dialects do this.